src/basic-languages/python/python.ts

```diff
@@ -250,16 +250,32 @@ export const language = <languages.IMonarchLanguage>{
 		// Recognize strings, including those broken across lines with \ (but not without)
 		strings: [
 			[/'$/, 'string.escape', '@popall'],
-			[/f'{1,3}/, 'string.escape', '@fStringBody'],
+			[/f'{3}/, 'string.escape', '@fStringBody3'],
+			[/f'{1}/, 'string.escape', '@fStringBody1'],
 			[/'/, 'string.escape', '@stringBody'],
 			[/"$/, 'string.escape', '@popall'],
-			[/f"{1,3}/, 'string.escape', '@fDblStringBody'],
+			[/f"{3}/, 'string.escape', '@fDblStringBody3'],
+			[/f"{1}/, 'string.escape', '@fDblStringBody1'],
 			[/"/, 'string.escape', '@dblStringBody']
 		],
-		fStringBody: [
+		fStringBody3: [
+			[/[^\\'\{\}]+/, 'string'],
+			[/\{\{[^}]+\}\}/, 'string'],
+			[/\{f"{3}/, 'string', '@fDblStringBody3'], // for nested f-strings
+			[/\{f"{1}/, 'string', '@fDblStringBody1'], // for nested f-strings
+			[/\{[^\}'":!=]+/, 'identifier', '@fStringDetail'],
+			[/\\./, 'string'],
+			[/'''/, 'string.escape', '@popall'],
```

Comment:
Why was this added? The equivalent of this does not seem to be present in the fStringBody1 array. What use case does it fix?